pkg/collector/corechecks/snmp/internal/report/report_bandwidth_usage_test.go

```diff
@@ -335,10 +600,15 @@ func Test_metricSender_sendBandwidthUsageMetric(t *testing.T) {
 			},
 			expectedMetric: []Metric{
 				// ((5000000 * 8) / (160 * 1000000)) * 100 = 25.0
-				{"snmp.ifBandwidthInUsage.rate", 25.0},
```

Comment:
Was the function previously being mocked before?

Reply:
i'm interpreting as question re: having a previous sample mapped?

i mocked the map to have a previous sample to check that the config version was being picked up, and the expected metric emitted something - but the same idea can get across w/o a metric being emitted and checking that the map has updated with the new config's ifSpeed?